tap_anvil/schemas/submissions.json

```diff
@@ -94,7 +94,7 @@
         "string"
       ]
     },
-    "archivedAt": {
+    "completedAt": {
```

Comment:
is there a reason this is a string and not some date type?

Reply:
yeah previously we have it in a `anyOf: [null, datetime type]` kinda thing, but that was the reason we didn't have those columns in our dwh
https://github.com/svinstech/tap-anvil/pull/1